go.mod

```diff
@@ -2,8 +2,10 @@ module github.com/longhorn/types
 
 go 1.22.7
 
+replace google.golang.org/grpc => google.golang.org/grpc v1.68.1
```

Comment:
Do we still need this replace directive? since you already changed the version in the require directive.

Reply:
No, we can remove it.